src/js/utils/StringUtil.js

```diff
@@ -215,6 +215,29 @@ const StringUtil = {
     }, []);
 
     return this.idToTitle(splitID, splitBy, replace, removeConsecutive);
+  },
+
+  /**
+   * @param {String} str     - A string that we would like to trim.
+   * @param {String} trimStr - The string that we would like to trim from
+   *                         - the beginning and the end.
+   * @return {String}        - The trimmed string. If a non-string type is
+   *                         - entered it will just return the input.
+   */
+  lrtrim(str, trimStr) {
+    if (typeof str === "string") {
+      const defaultTrimCharacters = "\\s|\\uFEFF|\\xA0";
+
+      return str.replace(
+        new RegExp(
+          `^(${trimStr}|${defaultTrimCharacters})+|(${trimStr}|${defaultTrimCharacters})+$`,
```

Comment:
@bstavroulakis thanks for making some changes! However, I strongly believe we shouldn't trim both all white space AND a user defined char set. I think it is harder to reason. A sensible default that can be overridden is how I understand most `trim` utilities work.